lib/ansible/modules/network/ovs/openvswitch_db.py

```diff
@@ -102,8 +113,16 @@ def map_obj_to_commands(want, have, module):
             templatized_command = "%(ovs-vsctl)s -t %(timeout)s remove %(table)s %(record)s " \
                                   "%(col)s %(key)s=%(value)s"
             commands.append(templatized_command % module.params)
+        elif module.params['key'] is None:
```

Comment:
This check is confusing. why not have simple 'else'. Is there a path where have.keys() do not have 'key' and module.params['key'] is not None ? If no then putting simple else will make code much readable .

Reply:
Probably not, you're right.

Reply:
There is a case when you try to delete a record that is non-existent. So we need to keep this condition.